packages/documentation/src/stories/components/tooltip/tooltip.stories.ts

```diff
@@ -0,0 +1,77 @@
+import { Args, Meta, StoryObj } from '@storybook/web-components';
+import { BADGE } from '../../../../.storybook/constants';
+import { html } from 'lit';
+
+const meta: Meta = {
+  title: 'Components/Tooltip',
+  component: 'post-tooltip',
+  parameters: {
+    badges: [BADGE.NEEDS_REVISION],
+    controls: {
+      exclude: ['class'],
+    },
+  },
+  render,
+  args: {
+    placement: 'top',
+    background: 'bg-primary',
+    message: 'Hi there 👋',
+  },
+  argTypes: {
+    message: {
+      name: 'Message',
+      description: 'Tooltip content',
+      control: {
+        type: 'text',
+      },
+    },
+    background: {
+      name: 'Background',
+      defaultValue: 'bg-primary',
+      description:
+        'Background color of the tooltip. Choose the one that provides the best contrast in your scenario.',
+      control: {
+        type: 'radio',
+        labels: {
+          'bg-primary': 'Primary',
+          'bg-yellow': 'Post yellow',
+        },
+      },
+      options: ['bg-primary', 'bg-yellow'],
+    },
+    slot: {
+      name: 'Slot',
+      control: false,
+      description:
+        'The tooltip default slot accepts any <a href="https://developer.mozilla.org/en-US/docs/Glossary/Inline-level_content">inline content</a> like `<span>` or `<post-icon>`, but no interactive content like `<a>` or `<button>` and no block level content like `<div>` or `p`.',
+    },
+  },
+  // Provide some margin for the tooltip to fit comfortably inside the iFrame with overflow: hidden
+  decorators: [
+    story =>
+      html`
+        <div style="margin-block: 5rem;">${story()}</div>
+      `,
+  ],
+};
+
+function render(args: Args) {
+  // Just for fun
+  const message = args.background === 'bg-yellow' ? args.message.replace('👋', '👋🏿') : args.message;
```

Comment:
The dark icons contrast is not ideal (at least on windows) and not reflected in the control table.
![image](https://github.com/swisspost/design-system/assets/9716662/114fdf93-3355-4d18-aa05-f7273eda4753)

Try to use one of the other shades: https://emojipedia.org/de/winkende-hand-medium-til-lys-teint

Reply:
Maybe it was only an issue on my screen, but I couldn't clearly see the dark icon on the yellow background.
Anyway, the value of the message control (in the controls table) is still not updated when we switch the background from `Primary` to `Post yellow`.
But I think we can ignore it in this case.